Jellyfin.Api/BaseJellyfinApiController.cs

```diff
@@ -8,7 +8,10 @@ namespace Jellyfin.Api
     /// </summary>
     [ApiController]
     [Route("[controller]")]
-    [Produces(MediaTypeNames.Application.Json)]
+    [Produces(
+        MediaTypeNames.Application.Json,
+        MediaTypeNames.Application.Json + "; profile=\"CamelCase\"",
+        MediaTypeNames.Application.Json + "; profile=\"PascalCase\"")]
```

Comment:
Will these strings be allocated every time a controller is initialized? Either way, they should probably be const.